ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/StateProcessor.java

```diff
@@ -194,6 +195,10 @@ public void onNewSlot(Date date) throws StateTransitionException, InterruptedExc
         new HeadStateEvent(
             BeaconStateWithCache.deepCopy((BeaconStateWithCache) headState), headBlock));
     recordData(date);
+    System.out.println(
```

Comment:
Why use `System.out.println` over `LOG.log`?

Reply:
I gotta clean that. Good catch 🤜